src/selectedtests/task_mappings/mappings.py

```diff
@@ -38,6 +38,7 @@ def create_task_mappings(
         file_regex: Pattern,
         module_name: str,
         module_file_regex: Pattern,
+        build_regex: Pattern,
```

Comment:
Could we set the default here to `'!.*'`? That way if someone calls `TaskMappings.create_task_mappings` directly, a default value is set.

It would also be good to test that `TaskMappings.create_task_mappings` succeeds even when `build_regex` is not passed in.

Reply:
I've removed the default from click and added the default in to ```create_task_mappings```